thirdparty/cmdconfig/commands/fnevalcmd.go

```diff
@@ -252,6 +259,7 @@ func (r *EvalFnRunner) preRunE(c *cobra.Command, args []string) error {
 		LogSteps:      r.LogSteps,
 		Env:           r.Env,
 		AsCurrentUser: r.AsCurrentUser,
+		FnConfigPath:  r.FnConfig,
```

Comment:
Read my earlier comment about naming it `FnConfigPath`. That will make this consistent.